test/clj/clojurewerkz/statistiker/metrics_test.clj

```diff
@@ -0,0 +1,49 @@
+(ns clojurewerkz.statistiker.metrics-test
+  (:require [clojurewerkz.statistiker.metrics :refer :all]
+            [clojurewerkz.statistiker.utils :refer [almost=]]
+            [clojure.test :refer :all]))
+; Tolerance for values
+(def tol 1e-8)
+
+(deftest wrong-input-mi
+  (testing "invalid inputs for mutual_information"
+    (is (thrown? AssertionError (mutual_information [1 2 3] [2 3])))
```

Comment:
If you have a bunch of `clojure.test/is` macros with basically identical structure you may want to consider using `clojure.test/are`.